save-frontend/src/main/kotlin/org/cqfn/save/frontend/components/views/ProjectView.kt

```diff
@@ -78,44 +156,121 @@ external interface ProjectViewState : RState {
  */
 @OptIn(ExperimentalJsExport::class)
 @JsExport
+@Suppress("CUSTOM_GETTERS_SETTERS")
 class ProjectView : RComponent<ProjectProps, ProjectViewState>() {
```

Comment:
Also, I think in bootstrap you [can use](https://getbootstrap.com/docs/4.0/components/collapse/#via-data-attributes) `data-parent` attribute and make an accordion component with automatically collapsing parts.

Reply:
Each modal opening call re render full page, that why we need to use state for all fields